### PR TITLE
[Documentation] Update README to reflect new CNTI program

### PR DESCRIPTION
**Pull request by wjsvec**
Related issue
https://github.com/lfn-cnti/bestpractices/issues/308

### CODE REVIEW

**Comment by wjsvec** [comment]
Take a preview here: 
https://github.com/wjsvec/cnf-wg/tree/wjs_dev_readme

**Review by electrocucaracha — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,12 +30,25 @@ The [CNF Test Suite](https://github.com/cncf/cnf-testsuite) will support testing
 
 ### Recurring meetings
 
-* Weekly on Mondays at 8:00am Pacific Standard Time / 16:00 UTC
+* **When**: Weekly on Mondays at 8:00am Pacific Standard Time / 16:00 UTC
   * Here's [a link](https://goo.gl/eyutah) to a public Google calendar that you can subscribe to
```
Comment (lixuna) [inline README.md]:
```suggestion
  * Here's [a link](https://zoom-lfx.platform.linuxfoundation.org/meeting/97973289576?password=167cfd8c-c026-4bac-8b4b-031be869659d) to register for future meetings
```

**Review by taylor — APPROVED** [review]
lgtm 👍🏻

File: README.md [hunk 2]
```diff
@@ -20,12 +30,25 @@ The [CNF Test Suite](https://github.com/cncf/cnf-testsuite) will support testing
...
-  * Passcode: :zero::four::zero::nine::six::three:
+* Join zoom meeting [here](https://join.slack.com/t/lfntech/shared_invite/zt-2cfymedlz-358~927JZBYfVJRMA7P9jg)
+  * **Meeting ID**: 97973289576
+  * **Meeting Passcode**: :five::four::nine::one::one::zero:
+  * **International numbers**: https://zoom.us/u/alwnPIaVT
+  
 * Agenda and notes are [available](https://docs.google.com/document/d/1YFimQftjkTUsxNGTsKdakvP7cJtJgCTqViH2kwJOrsc/edit)
-* Recordings of previous meetings: [CNF WG Playlist on YouTube](https://youtube.com/playlist?list=PLj6h78yzYM2PyMYvw5wiH01hthFb0qrOn)
+* Recordings of previous meetings: 
+
```
Comment (lixuna) [inline README.md]:
```suggestion
[Meeting Recordings](https://wiki.lfnetworking.org/display/LN/Certification)
```

File: README.md [hunk 3]
```diff
@@ -45,5 +68,8 @@ The [CNF Test Suite](https://github.com/cncf/cnf-testsuite) will support testing
 
 ## Community
 
-* Slack channel: [#cnf-wg](https://cloud-native.slack.com/archives/C01F1LVAQCC) in [CNCF workspace](https://slack.cncf.io/)
+* ### Slack
+
+  - [Join us on LFN Tech Slack](https://join.slack.com/t/lfntech/shared_invite/zt-2cfymedlz-358~927JZBYfVJRMA7P9jg)
+  - **Channel**: #cnti-bestpractices
 * Mailing list: <https://lists.cncf.io/g/cnf-wg>
```
Comment (lixuna) [inline README.md]:
```suggestion
* Mailing list: <https://lists.lfnetworking.org/g/lfn-cnti>
```

File: README.md [hunk 4]
```diff
@@ -20,12 +30,25 @@ The [CNF Test Suite](https://github.com/cncf/cnf-testsuite) will support testing
...
+
+   * [Jan 29, 2024](https://zoom.us/rec/play/Ps4s9hR4Nktk33S6L-hMCcIuBUON2K_UxZKYDdErDZLtT_wuI77XBnKpmRI1soVgdUo_HArPfkGQHXK8.aHMYV-3lgaD_uMBk?canPlayFromShare=true&from=share_recording_detail&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fzoom.us%2Frec%2Fshare%2F44Z7-aYAG5QCvH2w719qKv45L8x4ln9S6GsEs_WSh1U12tZEwAF_Ydab7R5VV_g-.TzWMrZCS_hEQ4Kk4)
+   
+   
+   - [Feb 5, 2024](https://zoom.us/rec/play/LF0uCZTgnbglWXPnrlw06-oWgmQ7qpXKh1PGRrpzOa3Te7OMUVGG8WI0jauLcz_OpM5M-Jzpi05qTbku.Exc5o4eQLFpdnPtv?canPlayFromShare=true&from=share_recording_detail&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fzoom.us%2Frec%2Fshare%2FXIdbBRkBBz_X-qIry8AOM63DuhGgJ4BNR4_YTM5N5pV5ZIXheIdtqvBWf2SKeaF6.CNzv8_J5AyjxjWi2)
+   
+   - [Feb 12, 2024](https://zoom.us/rec/play/YOazZV2bq6qtgK0yP_cujGjfFzxMBlp2PUhVnnj1KhlzmhEJK2zvXTNtpiGtbA_eHaVFgNnV5HItnWMH.C2iLtHxkmiO7epad?canPlayFromShare=true&from=share_recording_detail&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fzoom.us%2Frec%2Fshare%2F-4H-b8CthLkItEQ48ms-mREVHAD-PwtiloYPVjBHz262odsSrmafbtRp9UYZWYdH.EHyHF7owi_D7idrs)
+
 
 ### Upcoming events
```
Comment (lixuna) [inline README.md]:
```suggestion
Wednesday, May 1 • 2:00pm - 2:30pm
[Evolving Together: Cloud Native Telecom's Journey Forward - Olivier Smith, MATRIXX (CNTI)](https://sched.co/1YUsR)
```

**Review by lixuna — APPROVED** [review]

**Pull request merged**